app/sagas/init.js

```diff
@@ -25,16 +28,30 @@ export const initLocalSettings = function* initLocalSettings() {
 
 const restore = function* restore() {
 	try {
-		const { token, server } = yield all({
-			token: UserPreferences.getStringAsync(RocketChat.TOKEN_KEY),
-			server: UserPreferences.getStringAsync(RocketChat.CURRENT_SERVER)
-		});
+		const server = yield UserPreferences.getStringAsync('currentServer');
```

Comment:
Why did you change from `UserPreferences.getStringAsync(RocketChat.CURRENT_SERVER)` to the string version?